scripts/provisioning/configcmd.py

```diff
@@ -244,11 +244,9 @@ def push_s3_ldap_schema(self):
       ldap_lock = False
       self.logger.info('checking for concurrent execution scenario for s3 ldap scheam push using consul kv lock.')
       openldap_key=self.get_confkey("S3_CONSUL_OPENLDAP_KEY")
-      # TODO : update protocol and port
-      consul_endpoint_url=self.get_endpoint("CONFIG>CONFSTORE_CONSUL_ENDPOINTs", "fqdn", "tcp")
-      consul_endpoint_port=8500
+      consul_endpoint_url=self.get_endpoint("CONFIG>CONFSTORE_CONSUL_ENDPOINTS", "fqdn", "http")
```

Comment:
Use a default value here and we merge the same

Reply:
done

Reply:
done